src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -1,41 +1,27 @@
 <template>
-<div id="conditionArea"
+<div v-if="condition"
      class="c-cs-editui__conditions"
-     :class="['widget-condition', { 'widget-condition--current': isCurrent }]"
+     :class="['widget-condition', { 'widget-condition--current': currentConditionIdentifier && (currentConditionIdentifier.key === conditionIdentifier.key) }]"
```

Comment:
We should change this to use `openmct.objects.makeKeyString(<identifier>) `at some point.

Reply:
Added TODO